src/sentry/models/sentryshot.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import uuid
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import Model
+from sentry.db.models.base import region_silo_only_model
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_only_model
+class SentryShot(Model):
+    """
+    Represents metadata for a screenshot within Sentry.io
```

Comment:
sentry.io, or the general sentry product?

Reply:
sentry.io. The endpoint using this metadata will open `sentry_url` in sentry.io and takes a screenshot of the component with `component_identifier`.

Reply:
I guess what I mean is will this work in self-hosted and ST? If that's the case, we typically don't mention sentry.io

Reply:
oh! got it. Yeah it can work in self hosted. I'll update the text